src/tootsee.cr

```diff
@@ -1,4 +1,5 @@
 require "mastodon"
+require "json"
```

Comment:
🤔 where is this used

Reply:
It's used in `captioner.cr`. I'm not sure if it's better to put requires in the files that use them vs. put them here at top-level :/